feature-dapp-impl/src/main/java/io/novafoundation/nova/feature_dapp_impl/utils/tabs/RealBrowserTabService.kt

```diff
@@ -45,8 +52,10 @@ class RealBrowserTabService(
 
     init {
         activeSessions.setCallback {
```

Comment:
I thinnk we should rename this to `setOnSessionRemoved` or smth similar. `setCallback` sounds too generic